src/tags/object/RichText/view.js

```diff
@@ -83,7 +83,11 @@ class RichTextPieceView extends Component {
       normedRange.text = selectionText;
       normedRange.isText = item.type === "text";
       normedRange.dynamic = this.props.store.autoAnnotation;
-      item.addRegion(normedRange, this.doubleClickSelection);
+
+      const selectedAnnotation = window.Htx.annotationStore.selected.toNames.get(item.name).toJSON()?.filter(annotation => annotation.perregion && (annotation.preselectedValues?.length ?? 0) > 0);
```

Comment:
I think that would be fine, @hlomzik what are your thoughts on that as I used `Htx. annotationStore` after your suggestion but I believe technically both options are fine since both point to the same object. comes down to what direction we want to reproduce in the code going forward

Reply:
Yep, `Htx` is only for external usage. The fact that we use it in Area and Image is just a dirty hack I'd like to get rid of. It should have a better comment about it =(
Also using `selected` to get connected annotation is also a bad practice, I'm fixing this right now.
So just do the same as in other places of this file — `item.annotation`